tests/test_database.py

```diff
@@ -399,3 +399,49 @@ def test_write_solution_new_session(example_database:Database, example_subject:S
     solution = Solution(name="bleh", id='new_solution')
     example_database.write_session(example_subject, session)
     example_database.write_solution(session, solution)
+
+def test_get_photoscan_info(example_database:Database, tmp_path:Path): # TODO: Need to finish this!
+    subject_id = "example_subject"
+    session_id = "example_session"
+    photoscan_id = "example_photoscan"
+    photoscan_info = example_database.get_photoscan_info(subject_id, session_id, photoscan_id)
+    assert(photoscan_info["id"] == "example_photoscan")
+    assert(photoscan_info["name"] == "ExamplePhotoscan")
+    assert(Path(photoscan_info["model_abspath"]) == \
```

Comment:
Instead of asserting that the abspath is exactly this, we can just assert that it points to an existing file.
That way if we ever restructure where the data files are stored the test doesn't have to break